scripts/check-sidebar-lazy-layout.py

```
@@ -1,621 +1,280 @@
+        "SidebarWorkspaceTableCellView.swift": ("NSTableCellView", "NSHostingView", "rootView"),
+        "SidebarWorkspaceTableViewImpl.swift": ("NSTableView", "updateTrackingAreas", "otherMouseDown"),
+    }
+    for filename, markers in required.items():
+        source = sources_by_name.get(filename)
+        if source is None:
+            violations.append(f"missing required AppKit-list file {filename}")
+            continue
+        clean = neutralize_swift(source)
+        for marker in markers:
+            if marker not in clean:
+                violations.append(f"{filename} is missing required marker {marker}")
```

Ruff (0.15.21)

[warning] Use `list.extend` to create a transformed list

(PERF401)